### PR TITLE
the icecream update (small food refactor)

### PR DESCRIPTION
**Pull request by silicons**
starts prepping the codebase for food reworks by adding dynamic ice cream :)

### CODE REVIEW

**Comment by timothyteakettle** [comment]
looks good so far

File: code/modules/food/drinks/flask.dm [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/obj/item/reagent_containers/food/drinks/flask
+	name = "\improper Facility Director's flask"
```
Comment (TheLordME) [inline code/modules/food/drinks/flask.dm:2]: why is the baseline type the directors flask?

File: code/modules/food/drinks/misc.dm [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+	desc = "A small bottle full of glue. The orange tip calls to you, and the fluid inside is non-toxic... Should you?"
+	icon_state = "glue"
+	drop_sound = 'sound/items/drop/cardboardbox.ogg'
+	volume = 30
+	center_of_mass = list("x"=15, "y"=13)
+	start_reagent = /datum/reagent/drink/glue
+
+/obj/item/reagent_containers/food/drinks/glue/on_reagent_change()
+	..()
+	if(reagents.total_volume)
```
Comment (TheLordME) [inline code/modules/food/drinks/misc.dm:12]: Shouldnt there be a comparison?

File: code/modules/food/food/ice_cream.dm [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+	w_class = WEIGHT_CLASS_TINY
+	throw_force = 0
+	damage_force = 0
+	bitesize = 3
+
+	// :troll:
+	atom_flags = NOREACT
+
+	/// already bit into? no double dipping!
+	var/no_double_dipping = FALSE
```
Comment (TheLordME) [inline code/modules/food/food/ice_cream.dm:17]: bad name

**Review by TheLordME — APPROVED** [review]

File: code/modules/food/drinks/misc.dm (hunk 2 above)
Comment (silicons) [reply code/modules/food/drinks/misc.dm:12]: it's a copypaste not an add

File: code/modules/food/drinks/flask.dm (hunk 1 above)
Comment (silicons) [reply code/modules/food/drinks/flask.dm:2]: its a copypaste not an add i think

File: code/modules/food/drinks/bottle/dairy.dm [hunk 4]
```diff
@@ -0,0 +1,29 @@
+//* KEEP THIS FILE SORTED OR I WILL SPRINKLE PEPPER ON YOUR EYEBALLS *//
+//* SORT BY PATH, NOT BY NAME. *//
+
+/obj/item/reagent_containers/food/drinks/bottle/coconutmilk
```
[the remark this answers is not in the document]
Comment (silicons) [reply code/modules/food/drinks/bottle/dairy.dm:4]: is it fruit???

**Review by TheLordME — APPROVED** [review]

**Review by BlueWildrose — APPROVED** [review]

**Pull request merged**